src/getCoinCombination.test.js

```diff
@@ -7,4 +7,59 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
```

Comment:
I would suggest adding two more test, like in the checklist is displayed:
https://mate-academy.github.io/qa-program/checklists/unit_tests.html#first-two-tests-should-validate-that-function-is-declared-and-data-type-the-function-returns

Reply:
fixed